src/pages/Faq/components/FaqPageSlice.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface IFaqPageSliceProps {
+  header: string;
+  text: string;
+}
+
+export const FaqPageSlice = (props: IFaqPageSliceProps) => {
```

Comment:
actually, it's not a slice =)

slice is something that is connected with Redux
and in this case it's a component that might be called like FaqItem